### ISSUE TITLE
something is wrong with the websockit

### ISSUE THREAD

**Issue by AngelicDivine** [rivafarabi/deckboard#98]
sorry for this again TT^TT

but it seems when ever i try to add a audio device within the program it seems to crash the program well some what it freezes to the point i have to clear it in the task manager not sure what this could be making this do this but i am not sure i even did a fresh install 

again sorry about this

**Comment by rivafarabi** [comment]
You mean audio device control from obs?

**Comment by AngelicDivine** [comment]
yes it seems every time i go to it to add it to a board something happens and it freezes the hole program

**Comment by DerFabsee** [comment]
I just added multiple audio devices and I didn't experience any crash, freeze or something similar.

**Comment by AngelicDivine** [comment]
for some reason its doing it to mine not sure what tho

**Comment by rivafarabi** [comment]
I can't replicate the issue. But I'm keepibg this issue open if similar issue show up.

**Comment by AngelicDivine** [comment]
is there a way to fully clear the program off my pc even after uninstalling it?
cuz i heard that even uninstalling it will leave small bits of the program

**Comment by rivafarabi** [comment]
database.db file in C:/User/%USER%/deckboard is the only piece of program that is left after removal.

And for the record, is the error occured while the app connected to OBS?

**Comment by AngelicDivine** [comment]
yes it is
